<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="5" hostname="AliaksandrDryschyts" name="api.ApiTest" tests="9" failures="0" timestamp="2020-02-25T09:52:56 MSK" time="18.544" errors="1">
  <testcase name="registrationStepFour" time="1.870" classname="api.ApiTest">
    <skipped/>
  </testcase> <!-- registrationStepFour -->
  <testcase name="registrationStepFour" time="1.884" classname="api.ApiTest">
    <skipped/>
  </testcase> <!-- registrationStepFour -->
  <testcase name="registrationStepFour" time="1.835" classname="api.ApiTest">
    <error type="org.json.JSONException" message="JSONObject[&quot;id&quot;] not found.">
      <![CDATA[org.json.JSONException: JSONObject["id"] not found.
at org.json.JSONObject.get(JSONObject.java:520)
at org.json.JSONObject.getInt(JSONObject.java:694)
at api.ApiTest.registrationStepFour(ApiTest.java:152)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:204)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:377)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:138)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:465)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:451)
]]>
    </error>
  </testcase> <!-- registrationStepFour -->
  <testcase name="registrationStepFour" time="1.789" classname="api.ApiTest">
    <skipped/>
  </testcase> <!-- registrationStepFour -->
  <testcase name="registrationStepThree" time="3.566" classname="api.ApiTest"/>
  <testcase name="registrationStepFour" time="1.749" classname="api.ApiTest">
    <skipped/>
  </testcase> <!-- registrationStepFour -->
  <testcase name="registrationStepThree" time="3.715" classname="api.ApiTest">
    <skipped/>
  </testcase> <!-- registrationStepThree -->
  <testcase name="registrationStepTwo" time="0.249" classname="api.ApiTest"/>
  <testcase name="registrationStepOne" time="1.887" classname="api.ApiTest"/>
</testsuite> <!-- api.ApiTest -->
